content/en/community/_index.md

```diff
@@ -14,49 +61,61 @@ title: Community
+## Stay informed
 
-### [gRPC Java](https://github.com/grpc/grpc-java/labels/help%20wanted)
+Be the first to know! Follow these active channels for the gRPC announcements:
 
-For Android Java and Java.
+{{% cards "main_channels" %}}
 
-### [gRPC Go](https://github.com/grpc/grpc-go/labels/Status:%20help%20wanted)
+## Join the conversation
 
-For the Go implementation.
+The [grpc-io][] forum, mentioned above, is a great place to ask technical
+questions and engage in discussions with other gRPC developers.
 
-[More on how to contribute to gRPC Documentation.](/contribute/)
+The following channels, though less actively monitored, are also good places to
+exchange with developers and find answers to your gRPC questions:
 
-## Mailing list
+{{% cards "other_channels" %}}
 
-Any questions or suggestions? Just want to be in the loop of what is going on with the project? Join the [mailing list](https://groups.google.com/forum/#!forum/grpc-io).
+### Community meetings
 
-## Join the gRPC Ecosystem
+Get fresh news, and come talk with gRPC developers and other community members
+at one of our regular online community meetings.
 
-We have an organization for all valuable projects around gRPC in the [gRPC Ecosystem](https://github.com/grpc-ecosystem). The goal is to have all projects around gRPC (showing integrations with other projects or building utilities on top of gRPC) to be showcased here. If you have a new project you would like to add to gRPC Ecosystem, please fill up the [gRPC Ecosystem Project Request](https://docs.google.com/a/google.com/forms/d/119zb79XRovQYafE9XKjz9sstwynCWcMpoJwHgZJvK74/edit) form. Please read the [contribution guidelines](https://github.com/grpc/grpc-contrib/blob/master/CONTRIBUTING.md) for gRPC Ecosystem before submitting.
+{{% cards "meetup" %}}
 
-## Gitter Channel
+For meeting information and notes, see [gRPC Community Meeting Working
+Doc][grpcmeetings] ([bit.ly/grpcmeetings][grpcmeetings]).
 
-Join other developers and users on the [gRPC Gitter channel](https://gitter.im/grpc/grpc).
+## Contribute
 
-## Reddit
+Interested in contributing to gRPC code or documentation? To learn how, see
+[Contributing]({{< relref "contribute" >}}) and
+[grpc-community/CONTRIBUTING.md][].
```

Comment:
To-be-contributors should have only one page to visit; one of the two pages should be dropped (or stripped of most of it's content and direct readers to the other).